talker_agent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.7-alpine
+FROM python:2.7-alpine
```

Comment:
this doesn't look related to the commit / branch. next time separate such things